release_notes.md

```diff
@@ -0,0 +1,51 @@
+# Release Notes
+
+## Enhancements
+
+>**IMPORTANT**: This solution pack now requires FortiSOAR version `7.4.0` and later
+
+- Added the tag `CICD` to enhance searchability for the *Continuous Delivery* solution pack on Content Hub
+
+- Aligned the sequence of the task *Setup the Development Environment* with *Setup the Source Control for Production Environment*
+
+- Removed the dependency on the **Attachments** records to create export templates
+
+    - Added *Content and Settings Export Templates* in export template record
+
+- Added a new team *Content Development* to add users involved with content development in FortiSOAR
+
+### Playbook Enhancements
+
+- Added Pull Request (PR) title as an issue comment in *Create Pull Request* playbook
+
+- Pass input record to child playbook in The *Fetch Latest Changes* playbook
+
+- Renamed *Get User Confirmation* step to *Confirm Apply Latest Changes* in *Pull Latest Content from GitHub* playbook
+
+- Removed redundant *Update Import Job* and *Modify Configuration Option* steps from *Setup Development Environment* playbook
```

Comment:
Can we just say, Enhanced and optimised *Setup Development Environment* playbook?

Reply:
A detailed release note establishes trust and gives an impression of transparency, IMO.